README.md

```diff
@@ -65,25 +65,29 @@ sudo yum install gcc-c++ automake libtool pkgconfig boost-devel \
 To install on a FreeBSD system, use
 
 ```sh
-pkg install devel/git devel/autoconf devel/automake devel/gmake devel/libtool \
+pkg install devel/git devel/cmake
   textproc/expat2 graphics/geos graphics/proj databases/postgresql94-client \
   devel/boost-libs lang/lua52 devel/pkgconf
 ```
 
-Then you should be able to bootstrap the build system:
+CMake 2.8.7 is needed to build oslm2pgsql. First generate the makefiles in separate folder:
 
-    ./autogen.sh
+    mkdir build && cd build
+    cmake .. -G "Unix Makefiles"
 
-And then run the standard GNU build install:
+To build debug version with tests use additional options:
 
-    ./configure && make && make install
+    cmake .. -G "Unix Makefiles" -DCMAKE_BUILD_TYPE=Debug -DTEST_BUILD=ON
 
-Please see `./configure --help` for more options on how to control the build
-process.
+If some packages are not found, specify them directly. For example, on old Ubuntu you may need to add the following option:
```

Comment:
I'd rather not specify excess custom install information for EOL distros, and we need to work without any additional options on Ubuntu 14.04